_opensearch/install/docker.md

```diff
@@ -5,127 +5,165 @@ parent: Install OpenSearch
 nav_order: 3
 ---
 
-# Docker image
+# Why use OpenSearch with Docker?
 
-You can pull the OpenSearch Docker image from either Docker Hub or the public gallery hosted on AWS Elastic Container Registry (ECR).
+[Docker](https://www.docker.com/) greatly simplifies the process of configuring and managing your OpenSearch clusters. You can pull official images from [Docker Hub](https://hub.docker.com/u/opensearchproject) or [AWS Elastic Container Registry (ECR)](https://gallery.ecr.aws/opensearchproject/) and quickly deploy a cluster using [Docker Compose](https://github.com/docker/compose) and any of the sample Docker Compose files included in this guide. Experienced OpenSearch users can further customize their deployment by creating a custom Docker Compose file.
 
-From [Docker Hub](https://hub.docker.com/u/opensearchproject):
-```bash
-docker pull opensearchproject/opensearch:latest
-docker pull opensearchproject/opensearch-dashboards:latest
-```
-
-From [AWS ECR](https://gallery.ecr.aws/opensearchproject/):
-```bash
-docker pull public.ecr.aws/opensearchproject/opensearch:latest
-docker pull public.ecr.aws/opensearchproject/opensearch-dashboards:latest
-```
-
-To download a specific version of OpenSearch or OpenSearch Dashboards, modify the image tag (`latest`) to point to a valid version number. For example, `docker pull opensearchproject/opensearch:1.3.0` will download the image corresponding to OpenSearch 1.3.0.
-{: .note}
-
-To check available versions, see [Docker Hub](https://hub.docker.com/u/opensearchproject).
-
-When you download a new version of OpenSearch and OpenSearch Dashboards, you need to modify your `docker-compose.yml` file with the image version number that you downloaded. For example, to update your images to version 2.2.0, replace the following two lines in the YAML file: `image: opensearchproject/opensearch:2.2.0` and `image: opensearchproject/opensearch-dashboards:2.2.0`
-{: .note}
-
-OpenSearch images use `amazonlinux:2` as the base image. If you run Docker locally, set Docker to use at least 4 GB of RAM in **Preferences** > **Resources**.
+Docker containers are portable and will run on any compatible host that supports Docker (such as Linux, MacOS, and Windows). The portability of a Docker container offers flexibility over other installations methods, like [RPM]({{site.url}}{{site.baseurl}}/opensearch/install/rpm/) or a manual [Tarball]({{site.url}}{{site.baseurl}}/opensearch/install/tar/) installation, which both require additional configuration after downloading and unpacking.
 
+This guide assumes that you are comfortable working from the Linux command line interface (CLI). You should understand how to input commands, navigate between directories, and edit text files. For help with [Docker](https://www.docker.com/) or [Docker Compose](https://github.com/docker/compose), please refer to the official documentation on their websites.
```

Comment:
This is the same verbiage that we agreed on for the other guides. The guide does assume Linux CLI knowledge since all of our Docker images run on Linux.